### PR TITLE
Fix timezone in php

### PR DESCRIPTION
**Pull request by holzi1005**
## Summary by CodeRabbit

- **New Features**
	- Added a maintainer label and description to the Docker container.
	- Introduced a health check to verify application status.
	- Added a new environment variable `date_timezone` for PHP configuration.

- **Bug Fixes**
	- Updated PHP configuration to use dynamic variables for improved flexibility.

- **Documentation**
	- Enhanced `README.md` with new environment variable details and formatting adjustments for clarity.

### CODE REVIEW

**Comment by holzi1005** [comment]
Hi @erseco 
I added an ENV Option to Set the PHP date.timezone in the custom.ini to e.g. use the image in other countries and timezones

File: README.md [hunk 1]
```diff
@@ -168,6 +168,9 @@ You can define the next environment variables to change values from NGINX and PH
 | PHP8   | post_max_size           | 8M            | Max size of post data allowed. This setting also affects file upload. To upload large files, this value must be larger than upload_max_filesize. Generally speaking, memory_limit should be larger than post_max_size.                                 |
 | PHP8   | upload_max_filesize     | 2M            | Maximum size of an uploaded file.                                                                                                                                                                                                                      |
 | PHP8   | zlib_output_compression | On            | Whether to transparently compress pages. If this option is set to "On" in php.ini or the Apache configuration, pages are compressed if the browser sends an "Accept-Encoding: gzip" or "deflate" header.                                               |
+| PHP8   | date_timezone | UTC            | Set the Timezone of PHP                                               |
```
Comment (erseco) [inline README.md:171]: Try to align the spaces like the others, add more spaces between` | date_timezone `and` | UTC `to be aligned with the rest of the document.

File: Dockerfile [hunk 2]
```diff
@@ -95,4 +190,5 @@ ENV nginx_root_directory=/var/www/html \
     memory_limit=128M \
     post_max_size=8M \
     upload_max_filesize=2M \
-    zlib_output_compression=On
+    zlib_output_compression=On \
+    date_timezone=UTC
```
Comment (erseco) [inline Dockerfile:194]: Be careful, date_timezone="UTC" should be quoted

File: Dockerfile [hunk 3]
```diff
@@ -1,6 +1,30 @@
 ARG ARCH=
 FROM ${ARCH}alpine:3.20
 
+LABEL Maintainer="Ernesto Serrano <info@ernesto.es>" \
```
Comment (erseco) [inline Dockerfile:4]: I don't know what is changing here, can you explain, maybe is an error

File: Dockerfile [hunk 4]
```diff
@@ -79,6 +103,77 @@ EXPOSE 8080
 ENTRYPOINT ["/bin/docker-entrypoint.sh"]
```
Comment (erseco) [inline Dockerfile:103]: Duplicated ENTRYPOINT

File: Dockerfile [hunk 5]
```diff
@@ -79,6 +103,77 @@ EXPOSE 8080
...
 
 
+# Configure a healthcheck to validate that everything is up&running
```
Comment (erseco) [inline Dockerfile:106]: I don't know what is changing here, can you explain, maybe is an error

**Review by erseco — CHANGES_REQUESTED** [review]
First, thanks for submitting this, I will be pleased of adding it, but please, review the comments and do the changes before merging it! Thank you again :)

**Comment by holzi1005** [comment]
Somethings got messed up. I added a new pull request.

**Pull request closed without merging**